lib/gui/components/drive-selector/controllers/drive-selector.js

```diff
@@ -170,4 +170,85 @@ module.exports = function(
     });
   };
 
+  // This workaround is needed to avoid AngularJS from getting
+  // caught in an infinite digest loop when using `ngRepeat`
+  // over a function that returns a mutable version of an
+  // ImmutableJS object.
+  //
+  // The problem is that every time you call `myImmutableObject.toJS()`
+  // you will get a new object, whose reference is different from
+  // the one you previously got, even if the data is exactly the same.
+  /**
+   * @private
+   * @param {Function} func - function
+   * @param {Function} compare - function
+   * @returns {Function} function
+   */
+  const memoizeImmutableListReference = (func, compare = _.isEqual) => {
```

Comment:
Definitely! I was wondering where to put it, but now we have `lib/shared/utils.js` so that's solved.